src/node/internal/internal_dns.ts

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+// Copyright Joyent and Node contributors. All rights reserved. MIT license.
+
+import {
+  sendDnsRequest,
+  validateAnswer,
+  normalizeMx,
+  normalizeCname,
+  normalizeCaa,
+  normalizePtr,
+  normalizeNaptr,
+  normalizeNs,
+  normalizeSoa,
+  normalizeTxt,
+  normalizeSrv,
+  type MX,
+  type CAA,
+  type NAPTR,
+  type SOA,
+  type SRV,
+  type TTLResponse,
+} from 'node-internal:internal_dns_client';
+import { DnsError } from 'node-internal:internal_errors';
+import { validateString } from 'node-internal:validators';
+import * as errorCodes from 'node-internal:internal_dns_constants';
+
+export function getServers(): string[] {
+  throw new Error('Not implemented');
```

Comment:
A comment on these "Not implemented" stubs describing them a bit more would be helpful for future contributors. Are we not implementing them for a particular reason? Is there a path to implementing them? Links to the Node.js docs describing them, etc. Basically just something that would give enough context for someone coming into the project later to know what, if anything, to do with these.